library/core/src/array/mod.rs

```diff
@@ -176,6 +176,18 @@ where
     }
 }
 
+#[stable(feature = "try_from_mut_slice_to_array", since = "1.58.0")]
+impl<T, const N: usize> TryFrom<&mut [T]> for [T; N]
+where
+    T: Copy,
+{
+    type Error = TryFromSliceError;
+
+    fn try_from(slice: &mut [T]) -> Result<[T; N], TryFromSliceError> {
+        <Self>::try_from(slice.as_ref())
```

Comment:
nit: Using a reborrow could make the code a bit clearer (but it's just a nit).

```suggestion
        <Self>::try_from(&*slice)
```

Reply:
Thanks! It looks better. I applied the suggestion at ac083c6.